lib/v2/reuters/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (router) => {
+    router.get('/investigates', require('./investigates'));
+    router.get('/:navigation/:category?', require('./index'));
```

Comment:
I am still digesting what dose it mean.

Reply:
> I am still digesting what dose it mean.

Better to rename `lib/v2/reuters/index` to `lib/v2/reuters/navigation`. Or `lib/v2/reuters/common` since the _Navagation_ and _Author_ routes share it?

Reply:
_Author_ does share _Navigation_ , name it common is reasonable.